swift/WorkflowUI/Sources/Container/ContainerViewController.swift

```diff
@@ -13,6 +13,9 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
+
+#if os(iOS)
```

Comment:
Would `#if canImport(UIKit)` be more appropriate? Not sure how the os check would behave in a Catalyst world.

Reply:
Oh right, it would work here.

Reply:
Updated.